src/css/components/_layout.scss

```diff
@@ -86,7 +86,7 @@ body.vjs-full-window {
   /* Fix for IE6 full-window. http://www.cssplay.co.uk/layouts/fixed.html */
   overflow-y: auto;
 }
-.video-js.vjs-fullscreen {
+.vjs-full-window .video-js.vjs-fullscreen {
```

Comment:
I guess just moving the `position:fixed` will be less invasive though, it seems like the top/bottom/etc are for the full-window and not really for fullscreen, so, it's more correct to keep them here.
However, I tried this out on ios, android, firefox, chrome, safari, IE11, and IE8 and they all work fine.